### DISCUSSION TITLE
Creation of `berry_mapping`

### DISCUSSION THREAD

**Discussion by s-hadinger** [berry-lang/berry#216] [General]
Hi Skiars, I have created a set of functions to automate mapping of C functions to Berry. It is used for LVGL mapping. The code is here (I'm finishing the documentation).
https://github.com/arendst/Tasmota/tree/development/lib/libesp32/berry_mapping

Are you Ok for me to create a `berry_mapping` repository under `berry-lang` umbrella and move this code here?

**Comment by skiars** [comment]
I have no problem.

**Comment by s-hadinger** [comment]
Done: https://github.com/berry-lang/berry_mapping
